src/pretalx/orga/templates/orga/submission/feedbacks_list.html

```diff
@@ -0,0 +1,27 @@
+{% extends "orga/cfp/base.html" %}
+{% load i18n %}
+{% load rich_text %}
+
+{% block content %}
+    <h2 class="d-flex w-100 justify-content-between align-items-start">
+        {% trans "Event feedback" %}
```

Comment:
"Attendee feedback" is probably better, just in case we'll offer more general event feedback later.